src/LooseMode.js

```diff
@@ -0,0 +1,19 @@
+import { Component, options } from 'preact';
+
+export default function LooseMode() {}
+
+LooseMode.prototype = new Component();
+
+LooseMode.prototype.componentDidMount = function() {
+	let oldVnodeHook = options.vnode;
```

Comment:
Right now this re-installs the hook on every mount. Just needs a flag to flip once installed so it doesn't happen again:

```js
let oldVnodeHook = false;

LooseMode.prototype.componentDidMount = function() {
  if (oldVnodeHook===false) {
    oldVnodeHook = options.vnode;
    options.vnode = vnode => {
      // etc
    };
  }
};
```